content/docs/command-reference/config.md

```diff
@@ -91,6 +91,11 @@ This is the main section with the general config options:
   notify the user when a new version is available. Accepts values `true`
   (default) and `false`.
 
+- `core.autostage` - if enabled, DVC will automatically stage (`git add`)
+  modified DVC files as it updates them. The files will not be commited, and
+  user modified files will not be staged unless DVC then later modifies them.
+  Accepts values `true` and `false` (default).
```

Comment:
How about

``if enabled, DVC will automatically stage (`git add`) `.dvc` files(1) created or modified by DVC commands(2). The files will not be committed. Accepts values `true` and `false` (default).``

2 questions though:
1. Does this only affect `.dvc` files? (Should probably work for all metafiles including dvc.yaml and .lock) — see https://github.com/iterative/dvc/issues/4330#issuecomment-693603525
2. Which DVC commands dispay this behavior? Just `dvc add`? `dvc commit`? Etc. A full list would be useful.

Thanks

Reply:
Sure. That's simpler lol

1. This includes all modified files.
2. Any command that changes tracked files should do this?

It replaces the code in the hook to remind you to track files so any command that would make that output would stage the files for commit instead when this is enabled.

Reply:
Thanks.

1. OK great, let's use ``[DVC metafiles](/doc/user-guide/dvc-files-and-directories)` then, please.
2. it would be great to try it and double check which ones, and add a list with some of them, the more obvious ones a t least, like `add`, `run`, `import`, maybe `commit`.

Reply:
I still wonder what the exact full list of commands that use that hook is.